containers/daap-python-base/tests/unit/data_platform_logging_test.py

```diff
@@ -0,0 +1,155 @@
+import json
+
+from data_platform_logging import DataPlatformLogger, _make_log_dict
+from freezegun import freeze_time
+
+extra_input = {
+    "lambda_name": "test_lambda",
+    "data_product_name": "test_database",
+    "table_name": "test_table",
+    "image_version": "test",
+    "base_image_version": "test",
+}
+
+
+@freeze_time("2023-01-01")
+def test__make_log_dict():
```

Comment:
[Minor] I wouldn't normally bother testing this, as it's a private function that is already exercised by the logger tests below.

Other than this minor point these tests look useful though 👍🏻

Reply:
Did this one first, but you're right is covered by the last one so probably not needed